docs/integrations/code-conventions.md

```diff
@@ -828,6 +832,11 @@ The entry is composed of multiple components.
 * The `ReadableContentsFormat` dictates how to format the value passed to the `HumanReadable` field.
 * The `EntryContext` is the dictionary of context outputs for a given command. For more information see [Outputs](#outputs).
 * The `IndicatorTimeline` is an optional field (available from Server version 5.5.0 and up) . It is only applicable for commands that operate on indicators. It is a dictionary (or list of dictionaries) of the following format:
+* The `PollingCommand` is a command that'll run after `NextRun` seconds pass.
+* The `NextRun` is the next run time in seconds for the `PollingCommand`. The `PollingCommand` will be executed after this time has passed.
+* The `Timeout` is the timeout in seconds for a polling sequence. The polling sequence will reach a timeout when this time has passed. However, if a user provided an `execution-timeout`, it'll override the timeout specified by this field.
```

Comment:
```suggestion
* The `Timeout` argument is the timeout in seconds for a polling sequence. The polling sequence will reach a timeout when this time has passed. However, if a user has provided an `execution-timeout`, it will override the timeout specified by this field.
```